deploy/operator.yml

```diff
@@ -12,275 +12,471 @@ parameters:
   - name: SOURCE_NAMESPACE
     value: xjoin
   - name: RECONCILE_INTERVAL
-    value: "120"
+    value: '120'
   - name: VALIDATION_INTERVAL
-    value: "600"
+    value: '600'
   - name: VALIDATION_INTERVAL_INIT
-    value: "120"
+    value: '120'
   - name: VALIDATION_PERCENTAGE_THRESHOLD
-    value: "5"
+    value: '5'
   - name: VALIDATION_PERCENTAGE_THRESHOLD_INIT
-    value: "5"
+    value: '5'
   - name: VALIDATION_ATTEMPTS_THRESHOLD
-    value: "10"
+    value: '10'
   - name: VALIDATION_ATTEMPTS_THRESHOLD_INIT
-    value: "60"
+    value: '60'
   - name: VALIDATION_PERIOD_MINUTES
-    value: "60"
+    value: '60'
   - name: VALIDATION_LAG_COMPENSATION_SECONDS
-    value: "600"
+    value: '600'
   - name: CONNECT_CLUSTER
-    value: "xjoin-kafka-connect-strimzi"
+    value: xjoin-kafka-connect-strimzi
   - name: ES_CONNECTOR_TASKS_MAX
-    value: "50"
+    value: '50'
   - name: JENKINS_MANAGED_VERSION
-    value: "v1.0"
+    value: v1.0
   - name: KAFKA_TOPIC_PARTITIONS
-    value: "500"
+    value: '500'
   - name: KAFKA_TOPIC_REPLICAS
-    value: "2"
+    value: '2'
   - name: ES_CONNECTOR_CONFIG
-    value: '{
-      "tasks.max": "{{.ElasticSearchTasksMax}}",
-      "topics": "{{.Topic}}",
-      "key.ignore": "false",
-      "connection.url": "${env:ELASTICSEARCH_ENDPOINT}",
-      {{if .ElasticSearchUsername}}"connection.username": "{{.ElasticSearchUsername}}",{{end}}
-      {{if .ElasticSearchPassword}}"connection.password": "{{.ElasticSearchPassword}}",{{end}}
-      "type.name": "_doc",
-      "transforms": "valueToKey, extractKey, expandJSON, deleteIf, flattenList, flattenListString, renameTopic",
+    value: >-
+      { "tasks.max": "{{.ElasticSearchTasksMax}}", "topics": "{{.Topic}}",
+      "key.ignore": "false", "connection.url": "${env:ELASTICSEARCH_ENDPOINT}",
+      {{if .ElasticSearchUsername}}"connection.username":
```

Comment:
this won't prevent me from merging this but is there anyway to prevent these template conditionals from being split between lines? This formatting just makes it a little harder to read